doc/user_guide/user_guide.md

```diff
@@ -188,7 +254,7 @@ CREATE OR REPLACE TABLE <schema_name>.<table_name> (
 ```
 ### JSON preparation
 
-In case you want to add whole json document in one single column, (see **AS_JSON_DOC** on: [Optional
+In case you want to add the key json document in one single column, (see **AS_JSON_DOC** on: [Optional
```

Comment:
I did not touch this part of the readme so far as I wonder if it really makes sense to have the two separate sections about json and avro - with the new way of specifiying RECORD_FIELDS this seems wrong: In all environments (avro, string, text) it is possible to emit different columns. Even with json it is valid to emit both the full record and individual field (this can handle schema evolution situations when you don't want to lose any new fields added by the producer). So in the end the "table presentation" just depends on the `RECORD_FIELDS` statement and not on the record format